test/automation/src/sql/notebook.ts

```diff
@@ -132,6 +132,20 @@ export class Notebook {
 		});
 		await this.waitForResultsGone(cellIds);
 	}
+
+	async waitForTrustedElements(): Promise<void> {
+		await this.code.waitForElement('iframe');
```

Comment:
Aren't these going to wait for ANY iframe/embed/svg/dialog? Don't you want to scope the selector down to specifically this notebook instead?

Reply:
That's true, I'll make the selector more specific. Thanks!